web_datetime_picker_default_time/static/src/js/datetimepicker_service.esm.js

```diff
@@ -0,0 +1,295 @@
+                            } else {
+                                // If selecting the second value after the first:
+                                // - simply toggle the focus index
+                                originalCreate.state.focusedDateIndex =
+                                    originalCreate.state.focusedDateIndex === 1 ? 0 : 1;
+                            }
+                        }
+                    }
+
+                    hookParams.onChange?.(value);
+                };
+
+                const updateValueFromInputs = () => {
+                    const values = zipWith(
+                        getInputs(),
+                        ensureArray(originalCreate.state.value),
+                        (el, currentValue) => {
+                            if (!el) {
+                                return currentValue;
+                            }
+                            // CUSTOM: Check if the input is start date input
+                            const index = getInputs().indexOf(el);
+                            isEndDateInput = index === 1;
+                            const [parsedValue, error] = safeConvert("parse", el.value);
+                            if (error) {
+                                updateInput(el, currentValue);
+                                return currentValue;
+                            }
+                            return parsedValue;
+                        }
+                    );
+                    updateValue(
+                        values.length === 2 ? values : values[0],
+                        "date",
+                        "input"
+                    );
+                };
+
+                const getInput = (valueIndex) => {
+                    const el = getInputs()[valueIndex];
+                    if (el && hookParams.target.contains(el)) {
+                        return el;
+                    }
+                    return null;
+                };
+
+                let lastAppliedValue = null;
+                let inputsChanged = [];
+
+                const apply = () => {
+                    const valueCopy = deepCopy(originalCreate.state.value);
+                    if (areDatesEqual(lastAppliedValue, valueCopy)) {
+                        return;
+                    }
+
+                    inputsChanged = ensureArray(originalCreate.state.value).map(
+                        () => false
+                    );
+
+                    hookParams.onApply?.(originalCreate.state.value);
+                    lastAppliedValue = valueCopy;
+                };
+
+                const saveAndClose = () => {
+                    if (popoverService.isOpen) {
+                        // Apply will be done in the "onClose" callback
+                        popoverService.close();
+                    } else {
+                        apply();
+                    }
+                };
+
+                const onInputChange = (ev) => {
```

Comment:
Can we avoid re-defining methods that we don't want to override/change?

If the method is exactly the same as the original, then you can probably reference the one in `originalService`